### PR TITLE
Move styles to style-module

### PR DESCRIPTION
**Pull request by Scarygami**
Since external stylesheets are [deprecated](https://www.polymer-project.org/1.0/docs/devguide/styling.html#external-stylesheets).

### CODE REVIEW

File: google-signin.html [hunk 1]
```diff
@@ -7,11 +7,12 @@
 <link rel="import" href="../paper-material/paper-material.html">
 <link rel="import" href="../iron-flex-layout/iron-flex-layout-classes.html">
 <link rel="import" href="google-icons.html">
+<link rel="import" href="google-signin-styles.html">
 
 <dom-module id="google-signin">
-  <link rel="import" type="css" href="google-signin.css">
   <template>
-    <style is="custom-style" include="iron-positioning"></style>
```
Comment (Scarygami) [inline google-signin.html]: @jeffposnick  `is="custom-style"` isn't necessary/shouldn't be used inside of custom elements, so I removed it with this PR as well.

Comment (jeffposnick) [inline google-signin.html]: Thanks, @Scarygami!

File: google-signin.html [hunk 2]
```diff
@@ -7,11 +7,12 @@
...
+    <style include="google-signin-styles"></style>
+    <style include="iron-positioning"></style>
```
Comment (ebidel) [inline google-signin.html]: I believe you can combine, but double check that it works:

`<style include="google-signin-styles iron-positioning"></style>`

Comment (Scarygami) [inline google-signin.html]: @ebidel yup, that worked, updated and squashed the commits

**Comment by Scarygami** [comment]
Now that I/O is over, maybe someone can look at the PRs that accumulated for this element :)

**Comment by abdonrd** [comment]
LGTM!

**Comment by ebidel** [comment]
@addyosmani

**Comment by Scarygami** [comment]
Rebased against the current master to fix merge conflicts...

**Pull request merged**